src/Miunie.Logger/ConsoleLogger.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Miunie.Core.Logging;
+
+namespace Miunie.Logger
+{
+    public class ConsoleLogger : ILogger
+    {
+        public void Log(string message)
+        {
+            if (message.StartsWith("ERROR"))
```

Comment:
I feel like this should really be moved into a different method (LogError) for example. This sorta requires the user to already know that starting the log message with `ERROR` will format it differently.

Reply:
Absolutely :raised_hands: Thanks! :hearts: